{"priority": "critical", "kind": "bug", "repository": {"links": {"self": {"href": "data/repositories/osrf/gazebo.json"}, "html": {"href": "#!/osrf/gazebo"}, "avatar": {"href": "data/bytebucket.org/ravatar/{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}ts=1694483"}}, "type": "repository", "name": "gazebo", "full_name": "osrf/gazebo", "uuid": "{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}"}, "links": {"attachments": {"href": "data/repositories/osrf/gazebo/issues/2428/attachments_page=1.json"}, "self": {"href": "data/repositories/osrf/gazebo/issues/2428.json"}, "watch": {"href": "https://api.bitbucket.org/2.0/repositories/osrf/gazebo/issues/2428/watch"}, "comments": {"href": "data/repositories/osrf/gazebo/issues/2428/comments_page=1.json"}, "html": {"href": "#!/osrf/gazebo/issues/2428/models-inserted-before-a-log-starts-being"}, "vote": {"href": "https://api.bitbucket.org/2.0/repositories/osrf/gazebo/issues/2428/vote"}}, "reporter": {"display_name": "Louise Poubel", "uuid": "{5cfa2075-477b-4ded-bdb9-8d2479544ec4}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B5cfa2075-477b-4ded-bdb9-8d2479544ec4%7D"}, "html": {"href": "https://bitbucket.org/%7B5cfa2075-477b-4ded-bdb9-8d2479544ec4%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:6ff86fcb-b7ab-44a5-b8a6-f6d9cae8b6e8/7d903d90-c5ea-4182-b7ef-0d467e9e1c74/128"}}, "nickname": "chapulina", "type": "user", "account_id": "557058:6ff86fcb-b7ab-44a5-b8a6-f6d9cae8b6e8"}, "title": "Models inserted before a log starts being recorded are not recorded", "component": {"name": "util::Logging-Playback", "links": {"self": {"href": "data/repositories/osrf/gazebo/components/283119.json"}}}, "votes": 0, "watches": 1, "content": {"raw": "How to reproduce:\r\n\r\n1. Open Gazebo\r\n1. Insert some models, at least one of them should move, like the double pendulum\r\n1. Record a log\r\n1. Close Gazebo\r\n1. Start Gazebo playing back the log.\r\n\r\nThe models never show up and messages like these keep being printed:\r\n\r\n~~~\r\n[Err] [World.cc:2383] Unable to find model[double_pendulum_with_base]\r\n[Err] [World.cc:2383] Unable to find model[unit_box]\r\n~~~\r\n\r\nThis was probably caused by pull request #2608", "markup": "markdown", "html": "<p>How to reproduce:</p>\n<ol>\n<li>Open Gazebo</li>\n<li>Insert some models, at least one of them should move, like the double pendulum</li>\n<li>Record a log</li>\n<li>Close Gazebo</li>\n<li>Start Gazebo playing back the log.</li>\n</ol>\n<p>The models never show up and messages like these keep being printed:</p>\n<div class=\"codehilite\"><pre><span></span><span class=\"k\">[Err] [World.cc:2383] Unable to find model[double_pendulum_with_base]</span>\n<span class=\"k\">[Err] [World.cc:2383] Unable to find model[unit_box]</span>\n</pre></div>\n\n\n<p>This was probably caused by <a href=\"#!/osrf/gazebo/pull-requests/2608/process-insertions-and-deletions-on-gz-log\" rel=\"nofollow\" class=\"ap-connect-link\">pull request #2608</a></p>", "type": "rendered"}, "assignee": null, "state": "resolved", "version": {"name": "8.0", "links": {"self": {"href": "data/repositories/osrf/gazebo/versions/242745.json"}}}, "edited_on": null, "created_on": "2018-02-15T21:40:50.243514+00:00", "milestone": null, "updated_on": "2018-03-14T15:33:33.010533+00:00", "type": "issue", "id": 2428}